var/spack/repos/builtin/packages/ratel/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ratel(Package, CudaPackage, ROCmPackage):
+    """Extensible, performance-portable solid mechanics with libCEED and PETSc"""
+
+    homepage = "https://gitlab.com/micromorph/ratel"
+    git      = "https://gitlab.com/micromorph/ratel.git"
+
+    maintainers = ['jedbrown', 'jeremylt']
+
+    version('develop', branch='main')
+
+    depends_on('libceed')
+    depends_on('petsc')
```

Comment:
Probably needs something like this:
```suggestion
    depends_on('libceed@develop', when='@develop')
    depends_on('petsc@develop', when='@develop')
```